libs/features/mas/web-components/src/merch-datasource.js

```diff
@@ -52,75 +49,127 @@ const cardContent = {
         },
         ctas: { size: 'l' },
     },
+    'special-offers': {
+        name: {
+            tag: 'h4',
+            slot: 'detail-m',
+        },
+        title: {
+            tag: 'h4',
+            slot: 'detail-m',
+        },
+        backgroundImage: {
+            tag: 'div',
+            slot: 'bg-image',
+        },
+        prices: {
+            tag: 'h3',
+            slot: 'heading-xs',
+        },
+        description: {
+            tag: 'div',
+            slot: 'body-xs',
+        },
+        ctas: { size: 'l' },
+    },
```

Comment:
What if we use constants for the variant names instead of magic strings to avoid typos and make the code easier to maintain? i.e.
```suggestion
const VARIANTS = {
    CATALOG: 'catalog',
    AH: 'ah',
    CCD_ACTION: 'ccd-action',
    SPECIAL_OFFERS: 'special-offers',
};

const cardContent = {
    [VARIANTS.CATALOG]: {
        title: { tag: 'h3', slot: 'heading-xs' },
        prices: { tag: 'h3', slot: 'heading-xs' },
        description: { tag: 'div', slot: 'body-xs' },
        ctas: { size: 'l' },
    },
    [VARIANTS.AH]: {
        title: { tag: 'h3', slot: 'heading-xxs' },
        prices: { tag: 'h3', slot: 'heading-xs' },
        description: { tag: 'div', slot: 'body-xxs' },
        ctas: { size: 's' },
    },
    [VARIANTS.CCD_ACTION]: {
        title: { tag: 'h3', slot: 'heading-xs' },
        prices: { tag: 'h3', slot: 'heading-xs' },
        description: { tag: 'div', slot: 'body-xs' },
        ctas: { size: 'l' },
    },
    [VARIANTS.SPECIAL_OFFERS]: {
        name: { tag: 'h4', slot: 'detail-m' },
        title: { tag: 'h4', slot: 'detail-m' },
        backgroundImage: { tag: 'div', slot: 'bg-image' },
        prices: { tag: 'h3', slot: 'heading-xs' },
        description: { tag: 'div', slot: 'body-xs' },
        ctas: { size: 'l' },
    },
};
```